package.json

```diff
@@ -1,171 +1,174 @@
-    "@types/lodash.flatmap": "^4.5.6",
-    "@types/mixpanel-browser": "^2.38.0",
-    "@types/ms.macro": "^2.0.0",
-    "@types/multicodec": "^1.0.0",
-    "@types/node": "^13.13.5",
-    "@types/numeral": "^2.0.0",
-    "@types/qs": "^6.9.2",
-    "@types/react": "^18.0.15",
-    "@types/react-copy-to-clipboard": "^5.0.2",
-    "@types/react-dom": "^18.0.6",
-    "@types/react-google-recaptcha": "^2.1.5",
-    "@types/react-gtm-module": "^2.0.1",
-    "@types/react-helmet": "^6.1.2",
-    "@types/react-redux": "^7.1.16",
-    "@types/react-router-dom": "^5.0.0",
-    "@types/react-virtualized-auto-sizer": "^1.0.1",
-    "@types/react-window": "^1.8.5",
-    "@types/react-window-infinite-loader": "^1.0.6",
-    "@types/rebass": "^4.0.5",
-    "@types/recharts": "^1.8.23",
-    "@types/styled-components": "^5.1.25",
-    "@types/ua-parser-js": "^0.7.36",
-    "@types/wcag-contrast": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "^4",
-    "@typescript-eslint/parser": "^4",
-    "@uniswap/token-lists": "^1.0.0-beta.21",
-    "@walletconnect/web3-provider": "^1.5.0",
-    "@web3-react/core": "^6.0.9",
-    "@web3-react/fortmatic-connector": "^6.0.9",
-    "@web3-react/injected-connector": "^6.0.7",
-    "@web3-react/ledger-connector": "^6.1.1",
-    "@web3-react/portis-connector": "^6.0.9",
-    "@web3-react/trezor-connector": "^6.1.6",
-    "@web3-react/walletconnect-connector": "^7.0.2-alpha.0",
-    "@web3-react/walletlink-connector": "^6.2.13",
-    "ajv": "^6.12.3",
-    "cids": "^1.0.0",
-    "compression-webpack-plugin": "^6.1.1",
-    "copy-to-clipboard": "^3.2.0",
-    "eslint": "^7.11.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-better-styled-components": "^1.1.2",
-    "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-react": "^7.30.1",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "ethers": "^5.4.6",
-    "husky": "^8.0.1",
-    "inter-ui": "^3.13.1",
-    "jazzicon": "^1.5.0",
-    "lodash.flatmap": "^4.5.0",
-    "multicodec": "^2.0.0",
-    "multihashes": "^3.0.1",
-    "node-vibrant": "^3.1.5",
-    "polished": "^3.3.2",
-    "prettier": "^2.7.1",
-    "qs": "^6.9.4",
-    "react": "^18.2.0",
-    "react-device-detect": "^1.6.2",
-    "react-dom": "^18.2.0",
-    "react-error-overlay": "^6.0.9",
-    "react-feather": "^2.0.8",
-    "react-popper": "^2.2.3",
-    "react-redux": "^7.2.6",
-    "react-router-dom": "^5.0.0",
-    "react-scripts": "^4.0.3",
-    "react-spring": "^9.5.2",
-    "react-use-gesture": "^6.0.14",
-    "rebass": "^4.0.7",
-    "redux-localstorage-simple": "^2.4.0",
-    "styled-components": "^5.3.5",
-    "typescript": "^4.4.3",
-    "wcag-contrast": "^3.0.0",
-    "workbox-core": "^6.3.0",
-    "workbox-precaching": "^6.3.0",
-    "workbox-routing": "^6.3.0"
-  },
-  "resolutions": {
-    "@kyberswap/ks-sdk-core": "^0.0.16",
-    "trezor-connect": "8.1.15",
-    "react-error-overlay": "6.0.9"
+  "homepage": "/",
+  "license": "GPL-3.0-or-later",
```

Comment:
just curious, what's different between GPL and MIT?

Reply:
> GPL is more restrictive than the MIT license

https://stackoverflow.com/a/3903472/8153505